static/js/components/search/LearnerResult.js

```diff
@@ -12,8 +12,14 @@ export default class LearnerResult extends React.Component {
     result: { _source: SearchResult }
   };
 
+  static gradeAverageDisplay (program) {
+    return _.has(program, 'grade_average') && _.isNumber(program.grade_average) ?
+      `${program.grade_average}%` :
+      '-';
```

Comment:
I'm not sure about this method. What do you think about refactoring this along the following lines:

```js
const hasGrade = program => (
  _.has(program, 'grade_average') && _.isNumber(program.grade_average)
);
```

this could be either at the module level or an instance method. Then in the render call we'd do:

```js
<span className="percent">
  { hasGrade(program) ? `${program.grade_average}` : '-' }
</span>
```

Just an idea, we may not want the `hasGrade` function for anything else but the code is a little more modular this way.